includes/Modules/Sign_In_With_Google.php

```diff
@@ -53,52 +71,226 @@ final class Sign_In_With_Google extends Module implements Module_With_Assets, Mo
 	public function register() {
 		add_filter( 'wp_login_errors', array( $this, 'handle_google_auth_errors' ) );
 
-		add_action( 'login_form_google_auth', array( $this, 'handle_google_auth' ) );
+		add_action( 'login_form_google_auth', $this->get_method_proxy( 'handle_auth_callback' ) );
```

Comment:
Let's namespace this action as well.
```suggestion
		add_action( 'login_form_googlesitekit_auth', $this->get_method_proxy( 'handle_auth_callback' ) );
```

Reply:
Updated.